### PR TITLE
Added README.md

### PR DESCRIPTION
**Pull request by Syycorax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Omega robot
+--*In developpement*--
+
+This is a bot for our discord, it will include many features. 
+# Can I contribute ?
+Yes, you can contribute to this repo by creating a pull-request
```
Comment (quentinguidee) [inline README.md]: I don't think this part is necessary 🤔

File: README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# What are the features?
+
+ - [ ] Linking Epsilon  and Omega issues and pull request when they are called with #1e for Epsilon and #1 for Omega
+ - [ ]  Old comands like !hug
```
Comment (quentinguidee) [inline README.md]: The user doesn't know anything about this "famous old command hug"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ - [ ]  And more comming later!
```
Comment (quentinguidee) [inline README.md]:
```suggestion
 - [ ] Linking Epsilon  and Omega issues and pull request when they are called with #1e for Epsilon and #1 for Omega
 - [ ] Old comands like !hug
 - [ ] And more comming later! 
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Omega robot
+--*In developpement*--
```
Comment (quentinguidee) [inline README.md]: You could put a shield.io tag instead :)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+This is a bot for our discord, it will include many features.
```
Comment (quentinguidee) [inline README.md]: According to me, "it will include many features" is not a very useful information in a README.

**Review by quentinguidee — CHANGES_REQUESTED** [review]

**Comment by antoine-royer** [comment]
You can take ideas were raised in the discussion.
Screenshot :
CdC Omega-Robot

File: README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+</p>
+This is a bot for our two discord servers
+
+# How can I contribute ?
+
+You can contribute to this repo by forking the repo, clonning it, make your changes, commit ( we recommend committing to a new branch) and finally make a pull request.
+
+#  What are the features?
+
+
```
Comment (quentinguidee) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Linking Epsilon and Omega issues and pull request when they are called with #1e for Epsilon and #1 for Omega
+- [ ] Old comands like !hug (send a random hug gif)
+- [ ] And more comming later!
+
+# What it has to respect 
+- [ ] A clear documentation in `README.md` which contains the installation, configuration and running instructions
+- [ ] Being easily customisable and maybe later with Omega's website
+- [ ] Have a well thought-out separation for commands
+- [ ] Differentiate perfectly and cleanly the 2 servers
```
Comment (quentinguidee) [inline README.md]: Nothing to change, but I like this part!

File: README.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+# How can I contribute ?
+
+You can contribute to this repo by forking the repo, clonning it, make your changes, commit ( we recommend committing to a new branch) and finally make a pull request.
+
+#  What are the features?
+
+  
+
+- [ ] Linking Epsilon and Omega issues and pull request when they are called with #1e for Epsilon and #1 for Omega
+- [ ] Old comands like !hug (send a random hug gif)
```
Comment (quentinguidee) [inline README.md]:
```suggestion
- [ ] Commands like `!hug` (send a random hug gif)
```

I would rather not referencing the old bot in this new one.

File: README.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+
+#  Omega robot
+
+<p align="center">
+
+  <a href="https://github.com/Omega-Numworks/Omega/   issues"><img alt="Issues" src="https://img.shields.io/github/issues/Omega-Numworks/Omega-robot?style=for-the-badge" /></a>
+  <br/>
+  <a href="https://github.com/Omega-Numworks/Omega-Robot"> <img alt ="Status" src="https://img.shields.io/badge/Status-Under%20developpement-yellow?style=for-the-badge&logo=appveyor"> </a>
```
Comment (quentinguidee) [inline README.md]:
```suggestion
  <a href="https://github.com/Omega-Numworks/Omega-Robot"> <img alt ="Status" src="https://img.shields.io/badge/Status-Under%20developpement-yellow?style=for-the-badge"> </a>
```

I think it's better without this icon (?)

File: README.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+  <br/>
+  <a href="https://discord.gg/X2TWhh9"><img alt="Discord" src="https://img.shields.io/discord/663420259851567114?color=blue&labelColor=292929&label=chat%20-%20discord&logo=discord&style=for-the-badge" /></a>
+</p>
```
Comment (quentinguidee) [inline README.md]: Maybe try without the `<br/>` tags?

File: README.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+This is a bot for our two discord servers
```
Comment (quentinguidee) [inline README.md]:
```suggestion
This is a bot for our two discord servers.
```

**Review by quentinguidee — CHANGES_REQUESTED** [review]

File: README.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+  <a href="https://github.com/Omega-Numworks/Omega/   issues"><img alt="Issues" src="https://img.shields.io/github/issues/Omega-Numworks/Omega-robot?style=for-the-badge" /></a>
+  <br/>
+  <a href="https://github.com/Omega-Numworks/Omega-Robot"> <img alt ="Status" src="https://img.shields.io/badge/Status-Under%20developpement-yellow?style=for-the-badge"> </a>
+  <br/>
+  <a href="https://discord.gg/X2TWhh9"><img alt="Discord" src="https://img.shields.io/discord/663420259851567114?color=blue&labelColor=292929&label=chat%20-%20discord&logo=discord&style=for-the-badge" /></a>
+</p>
+This is a bot for our two discord servers.
+
+# How can I contribute ?
```
Comment (quentinguidee) [inline README.md]:
```suggestion
## How can I contribute ?
```

File: README.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+You can contribute to this repo by forking the repo, clonning it, make your changes, commit ( we recommend committing to a new branch) and finally make a pull request.
+
+#  What are the features?
```
Comment (quentinguidee) [inline README.md]:
```suggestion
##  What are the features?
```

File: README.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Linking Epsilon and Omega issues and pull request when they are called with #1e for Epsilon and #1 for Omega
+- [ ] Commands like `!hug` (send a random hug gif)
+- [ ] And more comming later!
+
+# What it has to respect
```
Comment (quentinguidee) [inline README.md]:
```suggestion
## What it has to respect 
```

File: README.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (quentinguidee) [inline README.md]:
```suggestion
```

**Review by quentinguidee — CHANGES_REQUESTED** [review]

File: README.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+#  Omega robot
+
+<p align="center">
+
```
Comment (quentinguidee) [inline README.md]:
```suggestion
```

**Review by quentinguidee — CHANGES_REQUESTED** [review]

**Review by quentinguidee — APPROVED** [review]

**Pull request merged**